internal/integration/spec.go

```diff
@@ -0,0 +1,114 @@
+// Package integration ...
+// Copyright 2019 New Relic Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+package integration
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path"
+	"regexp"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+
+	yaml "gopkg.in/yaml.v2"
+)
+
+const fileNameMatcher = `^prometheus_.*\.yml$`
```

Comment:
small comment here: some yaml files sometimes are created with `.yaml` extension so we could also include them automatically instead of forcing renaming to `.yml`